powershell-module/Az.Tools.Migration/Functions/Public/Find-AzUpgradeCommandReference.ps1

```diff
@@ -91,6 +90,14 @@ function Find-AzUpgradeCommandReference
     )
     Process
     {
+        # write warning if given azurerm version is not supported
+        $supportedAzureRmVersion = @('6.13.1', '6.13.2')
+        if (-not $supportedAzureRmVersion.Contains($FromAzureRmVersion)) {
+            Write-Error "AzureRm $FromAzureRmVersion is currently not supported. Supported AzureRm versions include '6.13.1', '6.13.2'." -ErrorAction Stop
+        } else {
+            $FromAzureRmVersion = '6.13.1'
```

Comment:
Does it make a differerence if we put 6.13.2 here? My impression is it doesn't.

Reply:
> Does it make a differerence if we put 6.13.2 here? My impression is it doesn't.

you are right, actually it should respect the value from user input for more precise telemetry.